config/application.rb

```diff
@@ -1,32 +1,20 @@
-require_relative "boot"
+require_relative 'boot'
```

Comment:
was un-linting this intentional? (i guess it's because it's now excluded but makes the diff look a little weird to revert it)

Reply:
It wasn't.